src/main/java/org/spongepowered/common/data/processor/multi/tileentity/FurnaceDataProcessor.java

```diff
@@ -48,6 +50,8 @@ public FurnaceDataProcessor() {
         super(TileEntityFurnace.class);
     }
 
+    private final Cause cause = Cause.source(SpongeImpl.getPlugin()).build();
```

Comment:
Perhaps store this `Cause` in `SpongeImpl` instead, rather than having the same thing in many places?